test/component/test_component.py

```diff
@@ -47,19 +49,28 @@ def get_msg_body_from_rabbit(self, channel):
         actual_msg = channel.basic_get(queue=RABBIT_QUEUE)
         return actual_msg[2].decode('utf-8')
 
-    def publish_to_pubsub(self, object_id):
+    def publish_to_pubsub(self, tx_id, case_id):
         publisher = pubsub_v1.PublisherClient()
 
-        topic_path = publisher.topic_path(RECEIPT_TOPIC_PROJECT_ID, RECEIPT_TOPIC_NAME)
+        try:
```

Comment:
as before: try/except not needed

Can be reduced down to just the topic_path... and data blocks.